ArduCopter/mode_systemid.cpp

```diff
@@ -221,23 +225,16 @@ void ModeSystemId::run()
 #endif
 
     } else {
-        // set xy speed and acceleration limits
-        pos_control->set_max_speed_accel_xy(500.0f, 250.0f);
-        pos_control->set_correction_speed_accel_xy(500.0f, 250.0f);
-
-        // set vertical speed and acceleration limits
-        pos_control->set_max_speed_accel_z(-get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
-
-        // process pilot inputs unless we are in radio failsafe
-        // this was split into two parts to allow pilot input to accomplish closed loop sweeps
         if (!copter.failsafe.radio) {
             // apply SIMPLE mode transform to pilot inputs
             update_simple_mode();
 
-            pilot_vel = get_pilot_desired_velocity(250.0f);
-
+            // convert pilot input to reposition velocity
+            // use half maximum acceleration as the maximum velocity to ensure aircraft will
+            // stop from full reposition speed in less than 1 second.
+            const float max_pilot_vel = wp_nav->get_wp_acceleration() * 0.5;
+            vel_correction = get_pilot_desired_velocity(max_pilot_vel);
```

Comment:
@lthall why are you adding a vel correction here?  I thought we decided that the pilot would not have any control over the position or velocity while in the mode.